### PR TITLE
AE-marine_transport

### PR DESCRIPTION
**Pull request by aeckland2696**
Added marine transport code and updated ipynb. Attached screenshot of model output (topo elevation) with SL = 1000, wave base = 10, diffusivity = 100, time step = 10, and run duration = 10e3.
Screen Shot 2022-04-13 at 9 45 10 AM

### CODE REVIEW

File: src/volcanic_island_simulator.py [hunk 1]
```diff
@@ -106,10 +161,13 @@ def update(self, dt):
         self.flow_router.run_one_step()
 
         # Apply fluvial erosion, transport, and deposition
+        self.space.run_one_step()
 
         # Switch boundaries back to full grid
+        self.grid.status_at_node[under_water] = self.grid.BC_NODE_IS_CORE
 
         # Apply submarine sediment transport
+        self.ssd.run_one_step(dt)
```
Comment (vaga3461) [inline src/volcanic_island_simulator.py:170]: all of your code related to adding the SimpleSubmarineDiffuser looks good to me!

File: volcanic_island_model_demo.ipynb [hunk 2]
```diff
@@ -141,7 +126,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": "3.7.7"
```
Comment (vaga3461) [inline volcanic_island_model_demo.ipynb:129]: I'm not sure how to review this volcanic_island_model_demo.ipynb file. I think Greg set all of this up and it looks like you've just pulled his most recent changes, but haven't edited anything in this file yourself. Is that correct?  Might want to request Greg's review on this one to make sure it's all good.

**Review by vaga3461 — COMMENTED** [review]
nice work, Abby! volcanic_island_simulator.py looks like it's ready to have some minor merge conflicts resolved, and then can be merged with base. 

if you made any changes to volcanic_island_model_demo.ipynb, I'd suggest requesting another review since I wasn't sure how to review the code I was looking at. However, if all of the changes made to that file were made by Greg, then you're probably fine to go ahead and merge it as well.

**Comment by gregtucker** [comment]
@aeckland2696 I resolved the conflicts, you should be ok to merge now.

**Comment by josiearcuri** [comment]
going ahead and merging your branch @aeckland2696 - i'm excited to add variable sea level to this!

**Review by josiearcuri — APPROVED** [review]

**Pull request merged**